contracts-bedrock/contracts/L2/L1Block.sol

```diff
@@ -66,8 +66,17 @@ contract L1Block {
 
         number = _number;
         timestamp = _timestamp;
-        basefee = _basefee;
```

Comment:
I think it is very confusing to track only our own averaged basefee, and not the actual basefee, while still calling it basefee.

Can we instead create a separate `averageBasefee` field, and track both?

Reply:
Related: the PR description is outdated, it still describes the averaging based on history instead of this new approach.

Reply:
Strongly agree with having `basefee` be real base fee and having a separate averaged fee. I would also ask that the type of average be explicitly named, e.g.,`basefeeRollingAverage`.